website/docs/language/resources/provisioners/remote-exec.mdx

```diff
@@ -48,8 +48,10 @@ resource "aws_instance" "web" {
 
 The following arguments are supported:
 
-* `inline` - This is a list of command strings. They are executed in the order
-  they are provided. This cannot be provided with `script` or `scripts`.
+* `inline` - This is a list of command strings. Strings are collected in order as
+  a single `.sh` script which is executed on the remote host. A default shell `#!` 
+  is used unless a `#!` string is the first command, such as `#!/bin/bash`. 
```

Comment:
> A default shell `#!` is used unless a `#!` string is the first command, such as `#!/bin/bash`.

I suggest changing this to: "A default shell is used unless a specific shell is specified as the first command, e.g. `#!/bin/bash`."

My concern with this language is that it presumes a linux-y environment on the remote host; presumably these could also be Windows or another OS that does not use this `sh`-like syntax.

Reply:
Makes sense, I have only used the ssh portion with Linux so I'm not sure how Windows env is affected. 

I will wait update the language to "A default shell is used unless a shell is specified as the first command, e.g. `#!/bin/bash`." pending the additional feedback.

Reply:
I like @crw feedback. I'd suggest one update to make sure it uses active voice:

The provisioner uses a default shell unless you specify a specific shell as the first command (e.g., `#!/bin/bash`).

Reply:
Sounds good, will adjust accordingly.